### ISSUE TITLE
Generate alt issue with screen reads

### ISSUE THREAD

**Issue by pi0** [nuxt/image#179]
`alt` attribute is auto generated so when hashing images we have search engine optimization but it makes issues with screen-readers (reported by @debs-obrien)

**Comment by debs-obrien** [comment]
When a screen reader comes across null alt text, it will completely skip over the image, without announcing its presence. If no alt attribute is present, the screen reader will read the file name for the image instead, which can be a major distraction to those using screen reading technology. so having the alt as the src is the same as having no alt as far as screen readers work and its not good practice. 

Suggestion is to auto generate an empty alt tag. Empty alt tags are only useful if image is decorative or shows content that is redundant to real text nearby. [more info](https://www.w3.org/WAI/tutorials/images/decision-tree/)

```html
alt=""
```
Of course adding an alt tag is more beneficial to users and search engines but adding the file name is not a great option. Especially when using providers like cloudinary who add on unique id's to the filename meaning screen readers will read out something like.

```html
green-cat_fx4zmm
```
Users should always add their own alt tag but generating an empty one can be useful for certain situations.

**Comment by tanc** [comment]
I agree, I found it strange when I saw auto-generated alt tags when I specify `alt=""`. I think a default of `alt=""` is good, or at least if the developer specified an empty alt it should be used.

**Comment by pi0** [comment]
Alt generation removed in main (to be released)
